ImageKitSample/Program.cs

```diff
@@ -0,0 +1,385 @@
+
+            imagekit.PurgeStatus("62e5778f31305bff3223b791");
+
+            #endregion
+
+            #region  Upload BY URI| Bytes | Base64
+
+
+            // Upload By URI
+            FileCreateRequest request = new FileCreateRequest
+            {
+                Url = new Uri(@"C:\test.jpg"),
+                FileName = "test.jpg"
+            };
+            ResponseMetaData resp1 = imagekit.Upload(request);
+
+
+            //Upload by bytes
+
+            string base64 = "iVBORw0KGgoAAAANSUhEUgAAAAUAAAAFCAYAAACNbyblAAAAHElEQVQI12P4//8/w38GIAXDIBKE0DHxgljNBAAO9TXL0Y4OHwAAAABJRU5ErkJggg==";
+
+            byte[] bytes = Convert.FromBase64String(base64);
+            FileCreateRequest ob = new FileCreateRequest
+            {
+                Bytes = bytes,
+                FileName = Guid.NewGuid().ToString()
+            };
+
+            List<string> tags = new List<string>
+            {
+                "Software",
+                "Developer",
+                "Engineer"
+            };
+            ob.Tags = tags;
+            ob.Folder = "demo1";
+            string customCoordinates = "10,10,20,20";
+            ob.CustomCoordinates = customCoordinates;
+            List<string> responseFields = new List<string>
+            {
+                "thumbnail",
+                "tags",
+                "customCoordinates"
+            };
+
+            ob.ResponseFields = responseFields;
+            JObject optionsInnerObject = new JObject
+            {
+                { "add_shadow", true }
+            };
+            JObject innerObject1 = new JObject
+            {
+                { "name", "remove-bg" },
+                { "options", optionsInnerObject }
+            };
+            JObject innerObject2 = new JObject
+            {
+                { "name", "google-auto-tagging" },
+                { "minConfidence", 10 },
+                { "maxTags", 5 }
+            };
+            JArray jsonArray = new JArray
+            {
+                innerObject1,
+                innerObject2
+            };
+
+            ob.Extensions = jsonArray;
+            ob.WebhookUrl = "https://webhook.site/c78d617f-33bc-40d9-9e61-608999721e2e";
+            ob.UseUniqueFileName = false;
+            ob.IsPrivateFileValue = false;
+            ob.OverwriteFile = false;
+            ob.OverwriteAiTags = false;
+            ob.OverwriteTags = false;
+            ob.OverwriteCustomMetadata = true;
+            JObject jsonObjectCustomMetadata = new JObject
+            {
+                { "test1", 10 }
+            };
+            ob.CustomMetadata = jsonObjectCustomMetadata;
+
+            ResponseMetaData resp2 = imagekit.Upload(ob);
+
+            //Upload by Base64
+            FileCreateRequest ob2 = new FileCreateRequest
+            {
+                Base64 = base64,
+                FileName = Guid.NewGuid().ToString()
+            };
+            ResponseMetaData resp = imagekit.Upload(ob2);
+
+
```

Comment:
Unnecessary new lines and uneven spacing throughout the file. It won't impress anyone trying to use our product.